test/functional/legacy/083_tag_search_with_file_encoding_spec.lua

```diff
@@ -0,0 +1,110 @@
+-- Tests for tag search with !_TAG_FILE_ENCODING.
+
+local helpers = require('test.functional.helpers')
+local insert, source, clear, expect, write_file = helpers.insert,
+  helpers.source, helpers.clear, helpers.expect, helpers.write_file
+
+local has_iconv = function()
+  return os.execute(helpers.nvim_prog .. ' -u NONE -i NONE --embed --cmd ' ..
+    [['if has("iconv")|q|else|cq|endif']]) == 0
+end
```

Comment:
At note about your comment from an older commit:

> If I reverse the check ( ... !has("iconv") ... ) I do not get a pending message 

That's because `0` is not false in lua. Only `false` and `nil` are false in lua. Sorry for your trouble.